Sources/AppCoinsSDK/Utils/Maze.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Maze.swift
+//
+//
+//  Created by Graciano Caldeira on 12/07/2024.
+//
+
+import Foundation
+import CommonCrypto
+
+internal class Maze {
+    
+    private let apiKey = "IaY8iJ0amBu1hc68cmQMlS9W"
+    private let type = "mrm0EaNLza04b7cz"
+    static let shared: Maze = Maze()
+    
+    private init() {}
+    
+    internal func getConnection(environment: [UInt8]) -> String? {
```

Comment:
This should be just 'get', not 'getConnection'. And the argument should be 'key', as environment is specific to this use case but does not generalize well.